include/boost/geometry/algorithms/detail/distance/interface.hpp

```diff
@@ -320,10 +321,10 @@ struct distance
 
 
 /*!
-\brief \brief_calc2{distance} \brief_strategy
+\brief Calculate the distance between two geometries \brief_strategy
 \ingroup distance
 \details
-\details \details_calc{area}. \brief_strategy. \details_strategy_reasons
+\details The free function distance calculates the distance between two geometries \brief_strategy. \details_strategy_reasons
```

Comment:
Shouldn't it read

> The free `distance` function...?

Reply:
It's from a template we use:
https://github.com/boostorg/geometry/blob/develop/doc/doxy/Doxyfile#L98

Reply:
Ah, indeed there is repetition. How about something like `The free function distance calculates the length of the shortest path between two geometries`?

Reply:
I think it's good, for hand-written sentences. But, for the template, I think the current template is acceptable.